Networking/Networking/Model/ShippingLabel/Packages/ShippingLabelCustomsForm.swift

```diff
@@ -3,14 +3,21 @@ import Codegen
 
 /// Represents customs info for a shipping label package
 ///
-public struct ShippingLabelCustomsForm: Equatable, GeneratedFakeable {
+public struct ShippingLabelCustomsForm: Hashable, Equatable, GeneratedFakeable {
     /// ID of the associated package.
     ///
     /// This is for identifying the package when inputing customs form only,
     /// no need for encoding and sending to remote.
     ///
     public let packageID: String
 
+    /// Name of the associated package.
+    ///
+    /// This is for identifying the package when inputing customs form only,
+    /// no need for encoding and sending to remote.
+    ///
+    public let packageName: String
+
```

Comment:
This seems like a reasonable way to keep track of the package name internally for use in the customs form, given how we're already handling other package details. 👍 

It'd be nice if we had a generic package model (not specific to custom or predefined packages) so we could more easily access the various properties of each package (ID, name, etc.) but that probably isn't worth the effort at this point.

Reply:
> It'd be nice if we had a generic package model

I agree! I think we can get back to that when working on the multi-package support feature because it'll be more necessary then.